tests/typecheck/test/test_client.yml

```diff
@@ -35,3 +34,76 @@
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+      response = client.head('foo', follow=False)
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+      response = client.head('foo', follow=True)
+      reveal_type(response.redirect_chain)  # N: Revealed type is "builtins.list[Tuple[builtins.str, builtins.int]]"
+      x: bool
+      response = client.head('foo', follow=x)
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+
+      response = client.trace('foo')
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+      response = client.trace('foo', follow=False)
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+      response = client.trace('foo', follow=True)
+      reveal_type(response.redirect_chain)  # N: Revealed type is "builtins.list[Tuple[builtins.str, builtins.int]]"
+      x: bool
+      response = client.trace('foo', follow=x)
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+
+      response = client.put('foo')
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+      response = client.put('foo', follow=False)
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+      response = client.put('foo', follow=True)
+      reveal_type(response.redirect_chain)  # N: Revealed type is "builtins.list[Tuple[builtins.str, builtins.int]]"
+      x: bool
+      response = client.put('foo', follow=x)
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+
+      response = client.patch('foo')
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+      response = client.patch('foo', follow=False)
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+      response = client.patch('foo', follow=True)
+      reveal_type(response.redirect_chain)  # N: Revealed type is "builtins.list[Tuple[builtins.str, builtins.int]]"
+      x: bool
+      response = client.patch('foo', follow=x)
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+
+      response = client.delete('foo')
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+      response = client.delete('foo', follow=False)
+      response.redirect_chain  # E: "_MonkeyPatchedWSGIResponse" has no attribute "redirect_chain"
+      response = client.delete('foo', follow=True)
+      reveal_type(response.redirect_chain)  # N: Revealed type is "builtins.list[Tuple[builtins.str, builtins.int]]"
+      x: bool
+      response = client.delete('foo', follow=x)
```

Comment:
Maybe we should allow `bool`? I think the main usecase for this is in unit-tests. They will be checked in all cases, but we can just make things harder for users by producing errors (and possibly false-positives).

Reply:
@sobolevn sorry, I cannot follow you. Or do you mean "not allow `bool`" instead? Only boolean literals?

Reply:
Sorry :(

I mean that using `bool` should not produce this error. Because users will get a unittest failure if it is not `True` anyway. And I don't want to have false-positives in test code.

Is it better now? 😊